server/polar/donation/schemas.py

```diff
@@ -117,3 +117,32 @@ class DonationStatisticsPeriod(Schema):
 
 class DonationStatistics(Schema):
     periods: Sequence[DonationStatisticsPeriod]
+
+
+class DonationSummary(Schema):
+    id: UUID4
+    amount: CurrencyAmount
+    message: str | None
+    donor: DonationOrganization | DonationUser | None
+    created_at: datetime.datetime
```

Comment:
Should this be public?

Reply:
I think the date is fine to keep in the API - but not necessarily show it in the UI.